### PR TITLE
Global config fonts

### PR DESCRIPTION
**Pull request by divyapalTatva**

### CODE REVIEW

File: CIR.Application/Services/GlobalConfiguration/GlobalConfigurationFontsServices.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+using System.Text;
+using System.Threading.Tasks;
+using CIR.Core.Entities.GlobalConfiguration;
+using CIR.Core.Interfaces.GlobalConfiguration;
+
+namespace CIR.Application.Services.GlobalConfiguration
+{
+    public class GlobalConfigurationFontsServices: IGlobalConfigurationFontsServices
+    {
+        private readonly IGlobalConfigurationFontsRepository _globalConfigurationFontsRepository;
```
Comment (GohilAmit) [inline CIR.Application/Services/GlobalConfiguration/GlobalConfigurationFontsServices.cs:14]: not use underscore

File: CIR.Data/Data/GlobalConfiguration/GlobalConfigurationFontsRepository.cs [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+using Dapper;
+using Microsoft.AspNetCore.Mvc;
+using System.Data;
+
+namespace CIR.Data.Data.GlobalConfiguration
+{
+	public class GlobalConfigurationFontsRepository : ControllerBase, IGlobalConfigurationFontsRepository
+	{
+		#region PROPERTIES
+		private readonly CIRDbContext _CIRDBContext;
```
Comment (GohilAmit) [inline CIR.Data/Data/GlobalConfiguration/GlobalConfigurationFontsRepository.cs:15]: not use underscore

File: CIR.Data/Data/GlobalConfiguration/GlobalConfigurationFontsRepository.cs [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+					{
+						globalConfigurationFontsList = connection.Query<GlobalConfigurationFonts>("spGetGlobalConfigurationFonts", null, commandType: CommandType.StoredProcedure).ToList();
+					}
+				}
+
+				return new JsonResult(new CustomResponse<List<GlobalConfigurationFonts>>() { StatusCode = (int)HttpStatusCodesAndMessages.HttpStatus.Success, Result = true, Message = HttpStatusCodesAndMessages.HttpStatus.Success.GetDescriptionAttribute(), Data = globalConfigurationFontsList });
+			}
+			catch (Exception ex)
+			{
+				return new JsonResult(new CustomResponse<Exception>() { StatusCode = (int)HttpStatusCodesAndMessages.HttpStatus.InternalServerError, Result = false, Message = HttpStatusCodesAndMessages.HttpStatus.InternalServerError.GetDescriptionAttribute(), Data = ex });
```
Comment (GohilAmit) [inline CIR.Data/Data/GlobalConfiguration/GlobalConfigurationFontsRepository.cs:49]: need to change catch return response

File: CIR.Data/Data/GlobalConfiguration/GlobalConfigurationFontsRepository.cs [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+                            parameters.Add("@FontFileName", globalConfigurationFont.FontFileName);
+                            parameters.Add("@IsDefault", globalConfigurationFont.IsDefault);
+                            parameters.Add("@Enabled", globalConfigurationFont.Enabled);
+
+                            result = connection.Execute("spCreateOrUpdateGlobalConfigurationFonts", parameters, commandType: CommandType.StoredProcedure);
+                        }
+                    }
+                    if (result != 0)
+                    {
+                        return new JsonResult(new CustomResponse<string>() { StatusCode = (int)HttpStatusCodesAndMessages.HttpStatus.Saved, Result = true, Message = HttpStatusCodesAndMessages.HttpStatus.Saved.GetDescriptionAttribute(), Data = string.Format(SystemMessages.msgDataSavedSuccessfully, "GlobalConfiguration Reason") });
```
Comment (GohilAmit) [inline CIR.Data/Data/GlobalConfiguration/GlobalConfigurationFontsRepository.cs:87]: need to change return response

File: CIR/Controllers/GlobalConfiguration/GlobalConfigurationFontsController.cs [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+using Microsoft.AspNetCore.Mvc;
+
+namespace CIR.Controllers.GlobalConfiguration
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class GlobalConfigurationFontsController : ControllerBase
+    {
+        #region PROPERTIES
+        private readonly IGlobalConfigurationFontsServices _globalConfigurationFontsServices;
```
Comment (GohilAmit) [inline CIR/Controllers/GlobalConfiguration/GlobalConfigurationFontsController.cs:14]: not use underscore

File: CIR/Controllers/GlobalConfiguration/GlobalConfigurationFontsController.cs [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+                try
+                {
+                    return await _globalConfigurationFontsServices.CreateGlobalConfigurationFonts(globalConfigurationFonts);
+                }
+                catch (Exception ex)
+                {
+                    return new JsonResult(new CustomResponse<Exception>() { StatusCode = (int)HttpStatusCodesAndMessages.HttpStatus.InternalServerError, Result = false, Message = HttpStatusCodesAndMessages.HttpStatus.InternalServerError.GetDescriptionAttribute(), Data = ex });
+                }
+            }
+            return new JsonResult(new CustomResponse<string>() { StatusCode = (int)HttpStatusCodesAndMessages.HttpStatus.BadRequest, Result = false, Message = HttpStatusCodesAndMessages.HttpStatus.BadRequest.GetDescriptionAttribute(), Data = SystemMessages.msgBadRequest });
```
Comment (GohilAmit) [inline CIR/Controllers/GlobalConfiguration/GlobalConfigurationFontsController.cs:62]: need to change return response

File: CIR.Application/Services/GlobalConfiguration/GlobalConfigurationFontsServices.cs (hunk 1 above)
Comment (divyapalTatva) [reply CIR.Application/Services/GlobalConfiguration/GlobalConfigurationFontsServices.cs]: Fixed

File: CIR.Data/Data/GlobalConfiguration/GlobalConfigurationFontsRepository.cs (hunk 2 above)
Comment (divyapalTatva) [reply CIR.Data/Data/GlobalConfiguration/GlobalConfigurationFontsRepository.cs:16]: Fixed

File: CIR.Data/Data/GlobalConfiguration/GlobalConfigurationFontsRepository.cs (hunk 3 above)
Comment (divyapalTatva) [reply CIR.Data/Data/GlobalConfiguration/GlobalConfigurationFontsRepository.cs]: Fixed

File: CIR.Data/Data/GlobalConfiguration/GlobalConfigurationFontsRepository.cs (hunk 4 above)
Comment (divyapalTatva) [reply CIR.Data/Data/GlobalConfiguration/GlobalConfigurationFontsRepository.cs]: Fixed

File: CIR/Controllers/GlobalConfiguration/GlobalConfigurationFontsController.cs (hunk 6 above)
Comment (divyapalTatva) [reply CIR/Controllers/GlobalConfiguration/GlobalConfigurationFontsController.cs:62]: Fixed

File: CIR/Controllers/GlobalConfiguration/GlobalConfigurationFontsController.cs (hunk 5 above)
Comment (divyapalTatva) [reply CIR/Controllers/GlobalConfiguration/GlobalConfigurationFontsController.cs]: Fixed

**Review by GohilAmit — APPROVED** [review]

**Review by jayk-patel-tatvasoft — APPROVED** [review]

**Review by mittalvyas1 — APPROVED** [review]

**Pull request merged**